src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -241,11 +226,22 @@ export class FileShareDetailComponent implements OnInit{
                         updatedAt: Utils.formatDate(item.updatedAt)
                     }
                     this.acls.push(acl);
+                    if(dialog){
+                        this.checkAclsIp.push(item.accessTo);
+                    }
                 }
             })
             this.selectedAcls = [];
         })
     }
+    userIpChnage(index){
```

Comment:
Fix typo

Reply:
I have just modified it. Please check it